addons/web/static/src/js/widgets/data_export.js

```diff
@@ -160,11 +160,9 @@ var DataExport = Dialog.extend({
         if (this.isCompatibleMode) {
             exportedFields.unshift({ name: 'id', label: _t('External ID') });
         }
-
-        framework.blockUI();
-        this.getSession().get_file({
-            url: '/web/export/' + exportFormat,
-            data: {
+        this.call('async', 'rpc', `${this.record.model} XLSX export`, {
```

Comment:
missing translation for job name (note: it does not work with string interpolation over string template, use `_.str.sprintf()` instead!)